### DISCUSSION TITLE
[Feature Request] Setting a default volume for all applications

### DISCUSSION THREAD

**Discussion by Vuxchen** [File-New-Project/EarTrumpet#885] [Ideas]
I would love to see a feature with which i can set a default volume for all apllications for example that all applications start with 33% volume as long as i dont set it to another value.

I play games like counterstrike on high volume and all other apps on 20-40% with the current way windows and eartrumpt work every time i start a new application i havent started before it defaults back to 100% volume and is unplasently loud. 

For now i have counterstrike set to 100% and turn down all other application manually but it would be way nicer to be able to have all applications default to a set value and just turn up the applications i want to be louder. I think that would be well recived by many people who experienced it themselfs how they rip off their headphones in a panic just because the application they just started was unbearably loud.

**Comment by Akxe** [comment]
This would also solve problems with app like Forza Horizon 4 that with every lunch restarted volume at 100%! :)

**Comment by RedSnt** [comment]
Someone did do a fork some years back with this feature, but it has not been updated for quite a while: https://github.com/LukasDoubleU/EarTrumpet/tree/all  
 
Would be an amazing addition because getting ones ears blown out when starting new games is the oldest and most annoying thing in the world.
